doc/development/html_themes/index.rst

```diff
@@ -221,6 +221,60 @@ If your theme package contains two or more themes, please call
    ``sphinx.html_themes`` entry_points feature.
 
 
+Styling with CSS
+----------------
+The **stylesheets** setting allows to add custom CSS files to a theme.
+
+The structure of the HTML elements and their classes are currently not a
+well-defined public API. Please infer them from inspecting the built HTML
+pages. While we cannot guarantee full stability, they tend to be fairly
+stable.
```

Comment:
I'd either suggestion a `caution` [admonition](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#admonitions-messages-and-warnings) here, or that we be brave and build consensus that we're OK with the classnames we've selected for this.

Reply:
Nope, sorry - I'd misunderstood this as being specific to search results, when it clearly is not.  That would lead me towards the admonition suggestion.